spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/DynamicConformanceJob.scala

```diff
@@ -318,6 +320,19 @@ object DynamicConformanceJob {
     }
   }
 
+  private def handleControlInfoValidation(): Unit = {
```

Comment:
You have this method written in two places. Can we only one of them in a single place? Maybe inside `ControlInfoValidation`?

Reply:
Talking with @benedeki and we thought we could leave this for when the refactoring of standardization and conformance

Reply:
Yes, we discussed this with Adrian.  I commented that the  original `ControlInfoValidation` was starting doding little too much. And from the options what and how to etxract. we though this to be the way to go. Also considered to put this logic to one object, not to duplicate code, but it felt like a too empty shell.